test/integration/buffer_accounting_integration_test.cc

```diff
@@ -101,9 +101,23 @@ class Http2BufferWatermarksTest
       buffer_factory_ = std::make_shared<Buffer::TrackedWatermarkBufferFactory>();
     }
     const HttpProtocolTestParams& protocol_test_params = std::get<0>(GetParam());
-    config_helper_.addRuntimeOverride("envoy.reloadable_features.http2_new_codec_wrapper",
-                                      protocol_test_params.http2_new_codec_wrapper ? "true"
-                                                                                   : "false");
+    switch (protocol_test_params.http2_implementation) {
+    case Http2Impl::Nghttp2:
+      config_helper_.addRuntimeOverride("envoy.reloadable_features.http2_new_codec_wrapper",
+                                        "false");
+      config_helper_.addRuntimeOverride("envoy.reloadable_features.http2_use_oghttp2", "false");
+      break;
+    case Http2Impl::WrappedNghttp2:
+      config_helper_.addRuntimeOverride("envoy.reloadable_features.http2_new_codec_wrapper",
+                                        "true");
+      config_helper_.addRuntimeOverride("envoy.reloadable_features.http2_use_oghttp2", "false");
+      break;
+    case Http2Impl::Oghttp2:
+      config_helper_.addRuntimeOverride("envoy.reloadable_features.http2_new_codec_wrapper",
+                                        "true");
+      config_helper_.addRuntimeOverride("envoy.reloadable_features.http2_use_oghttp2", "true");
+      break;
+    }
```

Comment:
This pattern seems to be needed for a few integration tests.
Can probably refactor to function in `HttpIntegrationTest`.

Reply:
Done.